### DISCUSSION TITLE
antd pro 6.0.0-beta.1 怎么解决qiankun静态资源404的？

### DISCUSSION THREAD

**Discussion by DemoPan** [ant-design/ant-design-pro#10040] [Q&A]
看网上解决方案是配置跨域，和修改publicPath

```javascript
// conifg.ts
export default defineConfig({
  // 省略其它配置
  devServer: {}, // 启动项目报错 -- Invalid config keys: devServer
  publicPath: '//localhost:3001/',
})
```
这个配置不行，首先配置文件不认devServer，然后publicPath配置了也没效果。

参考的是qiankun的常见问题：[为什么微应用加载的资源会 404？](https://qiankun.umijs.org/zh/faq#a-%E4%BD%BF%E7%94%A8-webpack-%E8%BF%90%E8%A1%8C%E6%97%B6-publicpath-%E9%85%8D%E7%BD%AE)

**Comment by AIEPhoenix** [comment]
最新模板只要配置了publicPath之后加载各种资源问题
